rosbag2_py/src/rosbag2_py/_storage.cpp

```diff
@@ -139,24 +190,38 @@ PYBIND11_MODULE(_storage, m) {
       std::vector<rosbag2_storage::TopicInformation>,
       std::string,
       std::string>(),
-    pybind11::arg("version"),
-    pybind11::arg("bag_size"),
-    pybind11::arg("storage_identifier"),
-    pybind11::arg("relative_file_paths"),
-    pybind11::arg("files"),
-    pybind11::arg("duration"),
-    pybind11::arg("starting_time"),
-    pybind11::arg("message_count"),
-    pybind11::arg("topics_with_message_count"),
-    pybind11::arg("compression_format"),
-    pybind11::arg("compression_mode"))
+    pybind11::arg("version") = 6,
+    pybind11::arg("bag_size") = 0,
+    pybind11::arg("storage_identifier") = "",
+    pybind11::arg("relative_file_paths") = std::vector<std::string>(),
+    pybind11::arg("files") = std::vector<rosbag2_storage::FileInformation>(),
+    pybind11::arg("duration") = std::chrono::nanoseconds{0},
+    pybind11::arg("starting_time") = std::chrono::time_point<std::chrono::high_resolution_clock>(std::chrono::nanoseconds{0}),
+    pybind11::arg("message_count") = 0,
+    pybind11::arg("topics_with_message_count") = std::vector<rosbag2_storage::TopicInformation>(),
+    pybind11::arg("compression_format") = "",
+    pybind11::arg("compression_mode") = "")
```

Comment:
@emersonknapp Could you please binding and default initialization for `pybind11::arg("custom_data")`?

https://github.com/ros2/rosbag2/blob/50e28cbab7e1cbe227b64a14b721793f6bd57216/rosbag2_storage/include/rosbag2_storage/bag_metadata.hpp#L56